### ISSUE TITLE
Scenario 2.1 - Sync Kubernetes secret to Vault ???

### ISSUE THREAD

**Issue by mvcaaa** [postfinance/vault-kubernetes#44]
I don't think it's possible - there is no implementation for this, am I correct?

Feature request: track Kubernetes secrets, update Vault secrets on k8s secret change.

**Comment by marcsauter** [comment]
No, it is not implemented. Our use case was, to keep the deployment independent from Vault. That's why we synchronize the secrets and do not request them from Vault during the deployment. It was essential for us, that secrets only can be changed in Vault with a personalized login.
